### ISSUE TITLE
 "Error Message" => "Invalid API call. Please retry or visit the documentation"

### ISSUE THREAD

**Issue by MujahidAbbas** [d1am0nd/alpha-vantage#8]
I keep getting this error for each method for Intraday, daily and search symbol methods. Is anyone facing this issue? please need guidance.

**Comment by ghost** [comment]
Yes I am getting invalid for pretty much anything other than "daily" ie "weekly" and "monthly" return:

` "Error Message" => "Invalid API call. Please retry or visit the documentation (https://www.alphavantage.co/documentation/) for TIME_SERIES_MONTHLY."`

**Comment by alfonsogarza** [comment]
The issue is that the intraday API requires an `interval` parameter, which is not passed.

I've hacked my local code so that the Stock's `intraday` API takes an extra parameter and it merges it with in the params. Happy to do a pull request but ideally we would want the API to take in some sort of enum or limited set of constants rather than any arbitrary interval string, since only a handful are supported.

**Comment by alfonsogarza** [comment]
The issue is that the intraday API requires an `interval` parameter, which is not passed.

I've hacked my local code so that the Stock's `intraday` API takes an extra parameter and it merges it with in the params. Happy to do a pull request but ideally we would want the API to take in some sort of enum or limited set of constants rather than any arbitrary interval string, since only a handful are supported.

**Comment by timoleonn** [comment]
Any update? Still having issues.
